visualization/app/codeCharta/util/deltaGenerator.ts

```diff
@@ -71,7 +72,8 @@ export class DeltaGenerator {
 
 				comparisonNode.fileCount = {
 					added: comparisonNode.type === NodeType.FILE ? 1 : 0,
-					removed: 0
+					removed: 0,
+					metricsChanged: 0
```

Comment:
Let's call it "changed". It's consistent with "added" and "removed" and we already know it's about "fileCount".

Reply:
"Changed" would mean exactly that: the file was changed, if any of the metrics is not zero. We compare a project with time1 to time2.

Reply:
I see where your confusion is coming from: we use 0 and 1 as a boolean value. These numbers are later on counted together to be visualized in the folder overview as `n number of files changed`.

Reply:
Well, as far as I can tell, a file could change without any metric changes (for example, a comment could be rewritten or a variable could be renamed).

Or do our metrics really cover _every_ possible change?

Reply:
For us our imported metrics are the only source of truth. Thus, anything unrelated should not matter.

Reply:
Yeah, I agree with you. Is it okay, if I leave a **small** comment for newcomers then?

Because I was quite bamboozled myself during this issue. I spend the first _hours_ wondering: "How can the frontend know whether a file was changed? The frontend does not seem to see the whole code, only the metric evaluations! What am I missing?" In the end, I chose this property name to help our future newcomers.

Reply:
Resolved with commit [642af10](https://github.com/MaibornWolff/codecharta/pull/2890/commits/642af106442ca181174404ac51e84bc67ef93050)